rosco-web/config/packer/install_packages.sh

```diff
@@ -1,25 +1,47 @@
 #!/bin/bash
 
-echo "deb_repo=$deb_repo"
+# Strip the first part to avoid credentials leaks
+echo "repository=$(echo $repository | sed s/^.*@//g)"
+echo "package_type=$package_type"
 echo "packages=$packages"
 echo "upgrade=$upgrade"
 
 # Strip leading/trailing quotes if present.
-deb_repo=`echo $deb_repo | sed 's/^"\(.*\)"$/\1/'`
+repository=`echo $repository | sed 's/^"\(.*\)"$/\1/'`
 
 # Support space-separated list of packages to install.
 # Strip leading/trailing quotes if present.
-packages=`echo $packages | sed 's/^"\(.*\)"$/\1/'`
+packages=`echo $packages | sed 's/^"\(.*\)"$/\1/' | sed 's/,/ /g'`
 
-# Only add the new source repository if deb_repo is set.
-if [[ "$deb_repo" != "" ]]; then
-  echo "deb $deb_repo" | sudo tee /etc/apt/sources.list.d/spinnaker.list > /dev/null
-fi
 
```

Comment:
it's now optional, also added the missing repository configuration in the rpm section